ui/app/AppLayouts/Wallet/popups/TokenSettingsModal.qml

```diff
@@ -14,7 +14,9 @@ ModalPopup {
     id: popup
     //% "Manage Assets"
     title: qsTrId("manage-assets")
-    
+    signal addCustomTokenTriggered()
+    signal tokenDetailsTriggered(string address, string address, string address, string address)
```

Comment:
This looks like this wants to be changed.

When we listen to `onTokenDetailsTriggered` and access the `address` scoped argument, which one will we get?

Reply:
done :D